services/meta/client.go

```diff
@@ -983,6 +983,12 @@ func snapshot(path string, data *Data) error {
 		return err
 	}
 
+	if _, err := os.Stat(file); err == nil {
```

Comment:
Is there a way to change this so the remove is only done on windows? On Linux and Mac, I would much prefer using rename to overwrite the previous file since it's supposed to be atomic, but it seems like that doesn't work on Windows.